content/os/Linux-Hardening.md

```diff
@@ -0,0 +1,259 @@
+
+The two main control systems are [SELinux](https://en.wikipedia.org/wiki/Security-Enhanced_Linux) (used on Android and Fedora) and [AppArmor](https://en.wikipedia.org/wiki/AppArmor).
+
+Fedora includes SELinux preconfigured with some policies that will confine [system daemons](https://en.wikipedia.org/wiki/Daemon_(computing)) (background processes). You should keep it in Enforcing mode.
+
+openSUSE gives the choice of AppArmor or SELinux during the installation process. You should stick to the default for each variant (AppArmor for [Tumbleweed](https://get.opensuse.org/tumbleweed/) and SELinux for [MicroOS](https://microos.opensuse.org/)). openSUSE’s SELinux policies are derived from Fedora.
+
+Arch and Arch-based operating systems often do not come with a mandatory access control system and that must be configured manually for either [AppArmor](https://wiki.archlinux.org/title/AppArmor) or [SELinux](https://wiki.archlinux.org/title/SELinux).
+
+Linux desktops don't usually include individual app confinement rules, unlike Android which sandboxes every application installed.
+
+### Making your own policies/profiles
+
+You can make your own AppArmor profiles, SELinux policies, Bubblewrap profiles, and [seccomp](https://en.wikipedia.org/wiki/Seccomp) blacklist to have better confinement of applications. This is an advanced and sometimes tedious task, so I won’t go into detail about how to do it here, but there are a few projects that you could use as reference.
+
+- Whonix’s [AppArmor Everything](https://github.com/Whonix/apparmor-profile-everything)
+- Krathalan’s [AppArmor profiles](https://github.com/krathalan/apparmor-profiles)
+- noatsecure’s [SELinux templates](https://github.com/noatsecure/hardhat-selinux-templates)
+- Seirdy’s [Bubblewrap scripts](https://sr.ht/~seirdy/bwrap-scripts)
+
+### Securing Linux containers
+
+If you’re running a server, you may have heard of Linux Containers, Docker. Containers are more common in server and development environments where individual apps are built to operate independently.
+
+[Docker](https://en.wikipedia.org/wiki/Docker_(software)) is one of the most common container solutions. It is **not** a proper sandbox, and this means that there is a large kernel attack surface. You can should the [Docker and OCI Hardening](/os/docker-and-oci-hardening/) guide to mitigate this problem.
+
+Another option is [Kata containers](https://katacontainers.io/), where virtual machines masquerade as containers. Each Kata container has its own Linux kernel and is isolated from the host.
+
+## Security Hardening
+
+![opensuse-computer.jpg](/images/opensuse-computer.jpg)
+
+### Firewalls
+
+A [firewall](https://en.wikipedia.org/wiki/Firewall_(computing)) may be used to secure connections to your system. If you’re on a public network, the necessity of this may be greater than if you’re on a local trusted network that you control. 
```

Comment:
It's never good to lax your security just because you "trust and control the network" when networking itself is inherently insecure. I don't agree with this part at all.

Reply:
Oh yeah you are right, this was one of the PG edits. I sure did not put it there.